_posts/2022-04-08-mfa-phase-1-rollout.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Making popular Ruby packages more secure
+layout: post
+author: 
+author_email: 
+---
+
+Attacks on the software supply chain are increasing. Our community has not gone unscathed. RubyGems has been affected by supply chain attacks in the past. It’s important for us to mitigate these risks as much as possible. Recommending stronger security practices like enabling multi-factor authentication (MFA) on high-impact packages will be a first step in improving security of the RubyGems ecosystem.
+
+Account takeovers are the second most common attack on software supply chains ([source](https://arxiv.org/abs/2002.01139)). The countermeasure against this type of attack is simple: enabling MFA. Doing so will help reduce the chances of a successful attack by 99.9 percent ([source](https://www.microsoft.com/security/blog/2019/08/20/one-simple-action-you-can-take-to-prevent-99-9-percent-of-account-attacks/)).
+
+Starting today, top-100 RubyGems package maintainers will begin to see reminders on the RubyGems command-line tool and website if MFA is not enabled on their accounts. Anybody who maintains a gem with more than 165 million downloads will see this recommendation.
```

Comment:
Adding date and month may be more precise instead of "today".

Reply:
top-100 and 165 mil+ downloads may not always be accurate. lets talk about download count first and then clarify that we are hoping to target *at least* top 100 gems by downloads.